scripts/install

```diff
@@ -143,6 +143,7 @@ cd $final_path; ynh_npm ci
 
 ynh_node_path=$ynh_node
 ynh_script_progression --message="ynh_node_path: $ynh_node_path" --weight=1
+ynh_script_progression --message="ynh_node_load_PATH: $ynh_node_load_PATH" --weight=1
```

Comment:
Actually, why do to add it as a text displayed to the user? It seems unnecessary.

Reply:
@tituspijean yup just a debug message. I still haven't quite got this to work. will see on this next test run if this fixes the problem.